### PR TITLE
ROX-18154: replace multikey store with single key

### PR DESCRIPTION
**Pull request by janisz**
## Description

It looks like there is no need for multi key stores in production code and the only
usage of multiple key is in `TestMultiKeyStruct`. 
This PR removes second key `filed2` and renames the struct to `TestStruct`.

## Checklist
- [ ] Investigated and inspected CI test results
- [ ] Unit test and regression tests added
- [ ] Evaluated and added CHANGELOG entry if required
- [ ] Determined and documented upgrade steps
- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

If any of these don't apply, please comment below.

## Testing Performed

CI

### CODE REVIEW

**Comment by janisz** [comment]
Current dependencies on/for this PR:
* master
  * **PR #6680** <a href="https://app.graphite.dev/github/pr/stackrox/stackrox/6680" target="_blank">Graphite</a> 
    * **PR #6681** <a href="https://app.graphite.dev/github/pr/stackrox/stackrox/6681" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/stackrox/stackrox/6681?utm_source=stack-comment).

**Review by connorgorman — APPROVED** [review]

File: proto/storage/test.proto [hunk 1]
```diff
@@ -137,14 +137,14 @@ message TestSingleUUIDKeyStruct {
...
-message TestMultiKeyStruct {
+message TestStruct {
   enum Enum {
     ENUM0 = 0;
     ENUM1 = 1;
     ENUM2 = 2;
   }
   string key1                  = 1 [(gogoproto.moretags) = 'sql:"pk,id" search:"Test Key"'];
-  string key2                  = 2 [(gogoproto.moretags) = 'sql:"pk" search:"Test Key 2,store,hidden"'];
+  string key2                  = 2 [(gogoproto.moretags) = 'search:"Test Key 2,store,hidden"'];
```
Comment (dashrews78) [inline proto/storage/test.proto:147]: Should we change this field name since it is no longer a key?

**Review by dashrews78 — APPROVED** [review]
one nit about the field name in the struct.

**Review by md2119 — APPROVED** [review]

File: proto/storage/test.proto (hunk 1 above)
Comment (janisz) [reply proto/storage/test.proto:147]: I tried to remove it but then protocheck fails. I think it's OK to keep that

**Review by rhybrillou — APPROVED** [review]

**Pull request merged**